filebeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -34,6 +34,31 @@ include::../include/config-option-intro.asciidoc[]
 
 include::../include/var-paths.asciidoc[]
 
+[float]
+==== `gc` log fileset settings
```

Comment:
On line 28 you see the fileset is defined. I assume we have to define it for each?

Reply:
No, that line is just so the config intro section on line 30 (rendered as "Variable Settings") uses the `server` fileset in it's text as an example.

Also, the content in `var-paths.asciidoc` doesn't depend on the `fileset_ex` variable anyway.

Altogether, the rendering looks like this:

<img width="579" alt="screen shot 2018-11-01 at 1 02 53 pm" src="https://user-images.githubusercontent.com/51061/47876395-7a560e00-ddd6-11e8-8c7e-8eda0b747291.png">